main/windows.js

```diff
@@ -83,14 +83,30 @@ function hideLauncherWindow() {
 
 function openAppWindow(app) {
     const lastWindowState = settings.loadLastWindow();
+
+    let { x, y } = lastWindowState;
+    const { width, height } = lastWindowState;
+    if (x && y) {
+        const { bounds } = electron.screen.getDisplayMatching(lastWindowState);
```

Comment:
Slightly better to use `workArea` instead of `bounds`, as bounds would include areas like a menu bar.

Reply:
The problem of workArea is that the window can be bigger.